pkg/subctl/cmd/gather/logs.go

```diff
@@ -62,41 +61,46 @@ func gatherPodLogsByContainer(podLabelSelector, container string, info Info) {
 	}
 }
 
-func podLogsToFile(pod *corev1.Pod, podLogOptions *corev1.PodLogOptions, info Info) error {
-	logRequest := info.ClientSet.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, podLogOptions)
+func getlogs(pod *corev1.Pod, podLogOptions *corev1.PodLogOptions, info Info) error {
```

Comment:
This should be camel-case but why was this renamed? `get` implies it retrieves and returns something which this function doesn't. This functions outputs the logs so the name should reflect that. We could rename to `outputPodLogs`.